addons/overheating/functions/fnc_jamWeapon.sqf

```diff
@@ -96,9 +104,9 @@ if (_unit getVariable [QGVAR(JammingActionID), -1] == -1) then {
             private _jamType = _one getVariable [format [QGVAR(%1_jamType), currentWeapon _one], "None"];
             private _jamMessage = localize LSTRING(FailureToFire);
             switch true do {
-                case (_jamType isEqualTo "Eject"): {_jamMessage = localize LSTRING(FailureToEject)};
-                case (_jamType isEqualTo "Extract"): {_jamMessage = localize LSTRING(FailureToExtract)};
-                case (_jamType isEqualTo "Feed"): {_jamMessage = localize LSTRING(FailureToFeed)};
+                case (_jamType isEqualTo "eject"): {_jamMessage = localize LSTRING(FailureToEject)};
+                case (_jamType isEqualTo "extract"): {_jamMessage = localize LSTRING(FailureToExtract)};
+                case (_jamType isEqualTo "feed"): {_jamMessage = localize LSTRING(FailureToFeed)};
```

Comment:
If I ain't missing something, doesn't look like this whole `switch true do` hack is even needed (plus it uses something like `isEqualTo` internally, since the comparisons are case-sensitive).

^ I think the whole `switch` statement could merge the above `private _jamMessage = localize` statement into a proper `private _jamMessage = switch _jamType do` so all these checks wouldn't be done for nothing; as much as I love `isEqualTo` for being pretty damn fast by SQF standards, an useless series of checks is an useless series of checks.

Reply:
Done!